packages/ra-ui-materialui/src/detail/TabbedShowLayout.js

```diff
@@ -77,36 +79,22 @@ export class TabbedShowLayout extends Component {
             translate,
             version,
             value,
+            tabs,
             ...rest
         } = this.props;
 
         return (
-            <div
-                className={className}
-                key={version}
-                {...sanitizeRestProps(rest)}
-            >
-                <Tabs
-                    // The location pathname will contain the page path including the current tab path
-                    // so we can use it as a way to determine the current tab
-                    value={location.pathname}
-                    indicatorColor="primary"
-                >
-                    {Children.map(children, (tab, index) => {
-                        if (!tab) return null;
-
-                        // Builds the full tab tab which is the concatenation of the last matched route in the
-                        // TabbedShowLayout hierarchy (ex: '/posts/create', '/posts/12', , '/posts/12/show')
-                        // and the tab path.
-                        // This will be used as the Tab's value
-                        const tabPath = getTabFullPath(tab, index, match.url);
-
-                        return cloneElement(tab, {
-                            context: 'header',
-                            value: tabPath,
-                        });
-                    })}
-                </Tabs>
+            <div className={className} key={version} {...sanitizeRestProps(rest)}>
+                {cloneElement(
+                    tabs,
+                    {
+                        // The location pathname will contain the page path including the current tab path
+                        // so we can use it as a way to determine the current tab
+                        value: location.pathname,
+                        match,
+                    },
+                    [...children],
```

Comment:
Why not put `children` directly here, instead of `[...children]`?

Reply:
Sorry, followed [this](https://reactjs.org/docs/react-api.html#cloneelement) blindly.